.github/workflows/build.yml

```diff
@@ -14,7 +14,7 @@ jobs:
       MAVEN_CACHE_KEY: ${{ secrets.MAVEN_CACHE_KEY }}
     strategy:
       matrix:
-        java: [ 11 ]
+        java: [ 11, 17 ]
```

Comment:
I am wondering whether it would be good to stick to a single Java version. If we specify more than one then that means we are restricted to the language features of the earliest one. I personally am interested in utilizing new language features.

Reply:
So, you're proposing updating to 17 when we touch things (and can do so without complications)?

Reply:
yes

Reply:
I'm good with this, too, if we land on the latest LTS as the release version. We can add newer JDKs as alternative test versions.

Reply:
Ah, I like that